<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiniPass</class>
 <widget class="QMainWindow" name="MiniPass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">minipass</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxOptions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="rowWrapPolicy">
       <enum>QFormLayout::RowWrapPolicy::DontWrapRows</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="removeCustomCharactersLabel">
        <property name="text">
         <string>Remove Custom Characters</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="removeCustomCharacters">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="makeMnemonicLabel">
        <property name="text">
         <string>Make Mnemonic</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="makeMnemonic">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="keepHistoryLabel">
        <property name="text">
         <string>Keep History</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="keepHistory">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="removeSpecialCharactersLabel">
        <property name="text">
         <string>Remove Special Characters</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="removeSpecialCharacters">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="removeUppercaseLettersLabel">
        <property name="text">
         <string>Remove Uppercase Letters</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="removeUppercaseLetters">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="removeLowercaseLettersLabel">
        <property name="text">
         <string>Remove Lowercase Letters</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="removeLowercaseLetters">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="removeNumbersLabel">
        <property name="text">
         <string>Remove Numbers</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="removeNumbers">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="defaultOptionsLabel">
        <property name="text">
         <string>Default Options</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="defaultOptions">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="passwordLength">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="passwordLengthLabel">
        <property name="text">
         <string>Password Length</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>511</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QPlainTextEdit" name="randomPassword">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="randomPasswordLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Random Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="logMessageLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Log Message</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="logMessage">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxActions">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Actions</string>
    </property>
    <widget class="QToolButton" name="generatePassword">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate Password</string>
     </property>
    </widget>
    <widget class="QToolButton" name="resetOptions">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset Options</string>
     </property>
    </widget>
    <widget class="QToolButton" name="exitMinipass">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Close Application</string>
     </property>
    </widget>
    <widget class="QToolButton" name="openPasswordsHistory">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Open PasswordsHistory.csv</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
